src/js/TreeView.js

```diff
@@ -7,6 +7,9 @@ function TreeViewItem(label) {
 
 	this.element = document.createElement('div');
 
+	// Store reference to itself. Needed for delegated mouseover
+	this.element.item = this;
```

Comment:
Brainstorming alternative solutions. Here is one idea.

1. In the top scope of our `TreeView` module, create a `WeakMap` (ex: `var scopeIdMap = new WeakMap()`) for storing a mapping of label element --> scope ID
2. In `TreeView.js`, inside the `TreeView.scopeItem` method, the `label` param refers to the scope id. Call `scopeIdMap.set`, with `item.element.firstChild`(label) as the key, and the `label` param (scope id) as the value
3. Inside the `mouseover` and `mouseout` event listeners, use `event.target` to get the label element. Then pass it to `scopeIdMap.get` to obtain the scope ID
4. Call `NGI.Scope.get` with the scope ID, which will return an ng-inspector scope object
5. On the Scope object, the `node` property will now refer to the scopes associated DOM element, that can be passed to the `Highlighter` module

We haven't used any `WeakMap`'s in the code-base yet, but [the basic syntax has relatively good support in Chrome/FF/Safari](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap)